wiki/Mapping_Techniques/Inspiration_Gallery_Corner/en.md

```diff
@@ -0,0 +1,176 @@
+# Inspiration Gallery Corner
+
+> blame naraku
+
+Here is a gallery of miscellaneous and interesting sliders.
+
+## narakucrimson's Sliders of Wonderment
+
+![](img/narakucrimson-1.png)
+
+![](img/narakucrimson-2.png)
+
+![](img/narakucrimson-3.png)
+
+## "U" Slider
+
+![](img/u.png)
+
+## Semi-Elbow
+
+![](img/semi-1.png)
+
+![](img/semi-2.png)
+
+## Awesome Wiggles
+
+![](img/awesome-1.png)
+
+![](img/awesome-2.png)
+
+## Interesting Heart-Shaped Slider
+
+![](img/interesting.png)
+
+## Slider Blankets
+
+![](img/slider-1.png)
+
+![](img/slider-2.png)
+
+![](img/slider-3.png)
+
+## Creativity is Just a Click Away!
+
+![](img/creativity-1.png)
+
+![](img/creativity-2.png)
+
+## Self-Blanketed Slider
+
+![](img/self-blanketed.png)
+
+## D33d's Pratting Around
+
+![](img/d33d-1.jpg)
+
+![](img/d33d-2.jpg)
+
+![](img/d33d-3.jpg)
+
+![](img/d33d-4.jpg)
+
+![](img/d33d-5.jpg)
+
+![](img/d33d-6.jpg)
+
+## BD's Big Fancy Things
+
+![](img/bd-1.jpg)
+
+![](img/bd-2.jpg)
+
+## Zodiac Signs
+
+> By Krisom
+
+### Capricorn
+
+![](img/capricorn.jpg)
+
+### Cancer
+
+![](img/cancer.jpg)
+
+### Gemini
+
+![](img/gemini.jpg)
```

Comment:
L87: Place a text inside ``[]`` to indicate which was supposed to be there if the image fails to load.